### DISCUSSION TITLE
Termux API question

### DISCUSSION THREAD

**Discussion by clubtech** [nathaneltitane/dextop#36] [General]
To clarify, are we supposed to install the Termux API APK before running the install script in Termux?
If yes, after installing the Termux API APK do we need to configure it in Termux before running the install script?

I am asking becase at the top of the instructions it states in install Termux, Termux API and Termux X11 but at the detailed instructions it only asks to install Termux and Termux X11

**Comment by nathaneltitane** [comment]
> To clarify, are we supposed to install the Termux API APK before running the install script in Termux? If yes, after installing the Termux API APK do we need to configure it in Termux before running the install script?
> 
> I am asking becase at the top of the instructions it states in install Termux, Termux API and Termux X11 but at the detailed instructions it only asks to install Termux and Termux X11

Termux: API is optional but still good to have - more updates to come that will leverage it.
The addons request can also take care of it for you if you agree to the install.

EDIT: to be fair, it's a very small APK and you're better off having it than not :)
